io500/sc23/aurora/aurora-reproducibility-productiony.md

```diff
@@ -0,0 +1,229 @@
+
+ALCF computing resources—available to researchers from academia, industry, and government agencies—support large-scale, computationally intensive projects aimed at solving some of the world's most complex and challenging scientific problems. Through awards of supercomputing time and support services, the ALCF enables its users to accelerate the pace of discovery and innovation across disciplines.
+
+Aurora will feature several technological innovations, including a revolutionary I/O system—the Distributed Asynchronous Object Store (DAOS)—to support new types of workloads. Aurora will be built on a future generation of Intel Xeon Scalable processor accelerated by Intel’s Xe compute architecture. Slingshot 11 fabric and HPE Cray EX supercomputer platform will form the backbone of the system. Programming techniques already in use on current systems will apply directly to Aurora. The system will be highly optimized across multiple dimensions that are key to success in simulation, data, and learning applications.
+
+[https://www.alcf.anl.gov/aurora]
+
+## AVAILABILITY
+
+> Please provide the deployment timeframe of the submitted system, or for on-demand cloud
+> systems, the general period over which it is deployed and destroyed.
+
+> Please describe the availability of the system to users and who are its set of most regular users.
+
+Aurora is available to scientific esearchers worldwide via three different allocations programs:
+  * The Innovative and Novel Computational Impact on Theory and Experiment (INCITE)
+  * ASCR Leadership Computing Challenge (ALCC)
+  * Director's Discretionary
+
+Aurora is expected to be available in 2024/2025.
+
+## STORAGE SYSTEM SOFTWARE
+
+> Please describe the purpose and general usage of the submitted system. This would include the
+> types of typical applications it supports (e.g., defense applications, molecular dynamics,
+> benchmarking, system test, systems research), the general use and purpose of the data generated
+> by the applications running on it.
+
+The Aurora DAOS storage system is available to any user on the Aurora system.
+Users are provided storage allocations based on their computing allocation requests.
+ 
+> How is this software available? (e.g., commercially, open-source, not publicly available) Note that if
+> the storage software is not open-source or commercially available, then a general description
+> would be requested, but this would limit the submission’s reproducibility score.
+
+> Can anyone download/purchase this software?
+
+> List either product webpage or open-source repo of the exact software used in IO500.
+
+The storage software is Distributed Asynchronous Object Store (DAOS), currently at Version 2.4.0-2.
```

Comment:
@mchaarawi may have a more precise commit hash for what he was running?

Reply:
I have that listed later in the document as 9597af34c30be0ea4049e43c16169057227e0527